AMW_rest/src/main/java/ch/mobi/itc/mobiliar/rest/deployments/DeploymentsRest.java

```diff
@@ -266,7 +267,7 @@ public Response getDeployments(
             createFiltersAndAddToList(DEPLOYMENT_PARAMETER, deploymentParameterValues, filters);
         }
 
-        Tuple<Set<DeploymentEntity>, Integer> result = deploymentBoundary.getFilteredDeployments(true, offset, maxResults, filters, null, null, null);
+        Tuple<Set<DeploymentEntity>, Integer> result = deploymentBoundary.getFilteredDeployments(false, offset, maxResults, filters, null, null, null);
```

Comment:
Can you do the same here as above instead of always disabling paging:
boolean doPageingCalculation = maxResults > 0